independent-projects/bootstrap/app-model/src/main/java/io/quarkus/bootstrap/model/PlatformImportsImpl.java

```diff
@@ -92,8 +94,10 @@ public void addPlatformProperties(String groupId, String artifactId, String clas
                         artifactId.length() - BootstrapConstants.PLATFORM_PROPERTIES_ARTIFACT_ID_SUFFIX.length()),
                 version);
         platformImports.computeIfAbsent(bomCoords, c -> new PlatformImport());
-        importedPlatformBoms.computeIfAbsent(groupId, g -> new ArrayList<>()).add(bomCoords);
-
+        importedPlatformBoms.computeIfAbsent(groupId, g -> new ArrayList<>());
+        if (!importedPlatformBoms.get(groupId).contains(bomCoords)) {
+            importedPlatformBoms.get(groupId).add(bomCoords);
+        }
```

Comment:
That's interesting. I didn't realize Maven and Gradle are different in this way.
@cdsap if it finds the `bomCoords` in the `importedPlatformBoms` we could skip the rest of the code in this method (reading and processing properties). Would you mind adjusting it? Thanks!